doc/api/fs.md

```diff
@@ -151,6 +151,10 @@ Stop watching for changes on the given `fs.FSWatcher`.
 
 Emitted when the ReadStream's file is opened.
 
+### Event: 'close'
+
+Emitted when the ReadStream's underlying file descriptor has been closed. Comes from fs.close() method. 
```

Comment:
The reason I ask is because either way we're wording it makes it kind of sound like the end user is expected to call `fs.close()`/`stream.close()` to see this event.